app/serializers/activitypub/follow_serializer.rb

```diff
@@ -1,11 +1,12 @@
 # frozen_string_literal: true
 
 class ActivityPub::FollowSerializer < ActiveModel::Serializer
-  attributes :id, :type, :actor
+  attributes :type, :actor
+  attribute :id, if: -> { object.respond_to(:object_type) }
```

Comment:
`respond_to?` (missing question mark)

I would also prefer using `if: :symbol` with a method definition as it's easier to read. E.g. `if: :supports_dereferencing?`